dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/message/MessageImpl.java

```diff
@@ -246,14 +292,9 @@ public static void copyContent(Message m1, Message m2) {
 
     @Override
     public void resetContextCache() {
-        if (contextCache != null) {
-            contextCache = null;
-        }
     }
 
     void setContextualProperty(String key, Object v) {
-        if (contextCache != null && !containsKey(key)) {
-            contextCache.put(key, v);
-        }
+        super.put(key, v);
```

Comment:
This does change behavior, but I believe it changes it to be accurate.  Before if there was no cache the setContextualProperty did nothing, and if there was a cache, a subsequent call to resetContextCache would cause the change to be lost.

Reply:
Also there should be a Liberty code change comment for this change.